### DISCUSSION TITLE
1panel拦截maxkb知识库上传

### DISCUSSION THREAD

**Discussion by Ausli** [1Panel-dev/1Panel#5865] [Q&A]
### 1Panel 版本

v1.10.12-lts

### 问题描述

见 https://github.com/1Panel-dev/MaxKB/issues/683

### 重现步骤

https://github.com/1Panel-dev/MaxKB/issues/683#issuecomment-2237944343
![image](https://github.com/user-attachments/assets/2e7dd9e2-a158-4ebd-b67a-50415ca429d6)

### 期待的正确结果

正常上传

**Comment by zhengkunwang223** [comment]
可以给这个 URL 加白名单
